packages/extension-base/ctp-extension-azure-adapters.spec.js

```diff
@@ -1,4 +1,4 @@
-const adapters = require('./ctp-extension-azure-adapters');
```

Comment:
Did this in two steps. I think we export multiple adapters if I understand your vision correctly.